pages.de/linux/pacman.md

```diff
@@ -0,0 +1,37 @@
+# pacman
+
+> Arch Linux Paket Management Tool.
+> Manche Unterbefehle wie `pacman sync` sind separat dokumentiert.
+> Weitere Informationen: <https://man.archlinux.org/man/pacman.8.de>.
+
+- Synchronisieren und alle Pakete aktualisieren:
+
+`pacman --sync --refresh --sysupgrade`
+
+- Ein neues Paket installieren:
+
+`pacman --sync {{paket_name}}`
```

Comment:
Now, I understand `Paketname` is a single word.
```suggestion
`pacman --sync {{paketname}}`
```

Reply:
also noticed that suchmuster is also a single word so I've fixed that in addition to this.